util/regressions.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tool for parsing a 'wpt run --log-wptreport' output file against a summary in
```

Comment:
This is cool. Just to look for funny things, or to be wrapped and somehow used in production?

Reply:
I think I've convoluted 2 different PRs here by mistake. Moved to https://github.com/w3c/wptdashboard/pull/311